Scottish-Tech-Army/Volunteer-app
# feat: create base container for profile screen

- [Description](#description)
  - [Type of change](#type-of-change)
- [Testing locally](#testing-locally)
- [Checklist](#checklist)

# Description

- Adds base container to build Profile Screen
- N.B. There was already a screen container called ProfileContainer, which contained the dark/light mode settings. This has been renamed to SettingsContainer. 

Fixes # ([SVA-352](https://github.com/Scottish-Tech-Army/Volunteer-app/pull/165))

## Type of change

Please delete options that are not relevant

- [] Bug fix (non-breaking change which fixes an issue)
- [] Breaking change(fix or feature that would cause existing functionality to not work as expected)
- [] This change requires a documentation update

# Testing locally

- check the Profile icon shows in the main nav bar at the bottom 
- check the screen opens and shows title centred 
- check it looks ok in in dark mode

# Checklist

- [X] I have addressed accessibility, if needed
- [] I have added tests that prove my fix is effective or that my feature works
